resources/definitions/atom2.def.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "Atom 2",
+  "version": 2,
+  "inherits": "fdmprinter",
+  "metadata": {
+    "visible": true,
+    "author": "Victor (Yu Chieh) Lin",
+    "manufacturer": "Layer One",
+    "file_formats": "text/x-gcode",
+    "platform_offset": [0,0,0],
+    "machine_extruder_trains":  { "0": "atom2_extruder_0"
+    }
+  },
+
+  "overrides": {
+    "machine_name":           { "default_value": "Atom 2" },
+    "machine_shape":          { "default_value": "elliptic" },
+    "machine_width":          { "default_value": 210 },
+    "machine_depth":          { "default_value": 210 },
+    "machine_height":         { "default_value": 320 },
+    "machine_extruder_count": { "default_value": 1 },
+    "machine_heated_bed":     { "default_value": false },
+    "machine_center_is_zero": { "default_value": true },
+
+    "machine_start_gcode":    { "default_value": "G21\nG90 \nM107\nG28\nG92 E0\nG1 F200 E3\nG92 E0" },
+    "machine_end_gcode":      { "default_value": "M104 S0\nG28\nG91\nG1 E-6 F300\nM84\nG90" },
+
+    "layer_height": { "default_value": 0.2 },
+    "default_material_print_temperature": { "default_value": 210 },
+    "speed_print": { "default_value": 32 },
+    "optimize_wall_printing_order": { "value": "True" },
+    "infill_sparse_density": { "default_value": 10 },
+    "brim_width":
+    {
+      "label": "Brim Width",
```

Comment:
Why are you overriding the Brim Width and Brim Line Count setting label/description/unit/etc?

If you want to adjust the minimum/maximum/etc. perhaps you should only adjust those.

As it stands, this would prevent us from making changes to these texts and properly translating those.

Reply:
I forgot where I saw this example, it was a copy and paste with a value change.
When I only have values that I want to change the profile wouldn't load, I'm not sure why.
But we could remove this segment, that's probably easier.

Reply:
I'd expect that changing it to this should work:

```
"brim_width": { "default_value": 4 },
```